veracruz-server/src/veracruz_server_linux.rs

```diff
@@ -12,109 +12,111 @@
 #[cfg(feature = "linux")]
 pub mod veracruz_server_linux {
 
+    use crate::{veracruz_server::VeracruzServer, VeracruzServerError};
+    use data_encoding::HEXLOWER;
+    use io_utils::{
+        http::{post_buffer, send_proxy_attestation_server_start},
+        tcp::{receive_message, send_message},
+    };
     use log::{error, info};
-
+    use policy_utils::policy::Policy;
+    use ring::digest::{digest, SHA256};
     use std::{
-        env,
+        fs::File,
+        io::Read,
         net::{Shutdown, TcpStream},
-        path::PathBuf,
         process::{Child, Command},
         thread::sleep,
         time::Duration,
     };
-
-    use crate::{veracruz_server::VeracruzServer, VeracruzServerError};
-    use io_utils::tcp::{receive_message, send_message};
-    use policy_utils::policy::Policy;
-    use veracruz_utils::platform::{
-        linux::{LinuxRootEnclaveMessage, LinuxRootEnclaveResponse},
-        vm::{RuntimeManagerMessage, VMStatus},
+    use transport_protocol::{
+        parse_proxy_attestation_server_response, serialize_native_psa_attestation_token,
     };
+    use veracruz_utils::platform::vm::{RuntimeManagerMessage, VMStatus};
 
     ////////////////////////////////////////////////////////////////////////////
     // Constants.
     ////////////////////////////////////////////////////////////////////////////
 
-    /// Path to the pre-built Linux root enclave.
-    const LINUX_ROOT_ENCLAVE_PATH: &'static str =
-        "../linux-root-enclave/target/release/linux-root-enclave";
-    /// Port to communicate with the Linux root enclave on.
-    const LINUX_ROOT_ENCLAVE_PORT: &'static str = "5021";
-    /// IP address to use when communicating with the Linux root enclave.
-    const LINUX_ROOT_ENCLAVE_ADDRESS: &'static str = "127.0.0.1";
+    /// Path of the Runtime Manager binary (the enclave).
+    const RUNTIME_ENCLAVE_BINARY_PATH: &'static str =
+        "../workspaces/linux-runtime/runtime_manager_enclave";
+    /// Spawn delay to apply (in seconds) between spawning the Runtime Manager enclave and trying
+    /// to contact it.
+    const RUNTIME_ENCLAVE_SPAWN_DELAY: u64 = 2;
     /// IP address to use when communicating with the Runtime Manager enclave.
     const RUNTIME_MANAGER_ENCLAVE_ADDRESS: &'static str = "127.0.0.1";
-    /// Delay (in seconds) to use when spawning the Linux root enclave to
-    /// ensure that everything is started before proceeding with communication
-    /// between the server and enclave.
-    const LINUX_ROOT_ENCLAVE_SPAWN_DELAY_SECONDS: u64 = 2;
+    /// Port to communicate with the Runtime Manager enclave on.
+    const RUNTIME_MANAGER_ENCLAVE_PORT: &'static str = "6000";
```

Comment:
Maybe we need to document the default port somewhere, e.g. README.md; or even provide a configuration option in the policy file.